README.adoc

```diff
@@ -100,8 +99,8 @@ guide is the following one:
 
 ==== Link types
 
-In the following example there are two different links. The first has a `self` relationship with the
-resource object and is generated whenever a host has been registered. The link points to that host
+The following example shows two different links. The first link has a `self` relationship with the
+resource object and is generated whenever you generate a host. The link points to that host
```

Comment:
Good change as well, but it should be "whenever you register a host", "generate" doesn't really fit into the context of the app

Reply:
Sure, we can go with "register" instead of "generate." :)